.editorconfig

```diff
@@ -0,0 +1,11 @@
+# http://EditorConfig.org
+
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
This is super useful. There's an editorconfig plugin for just about every editor. This configures your editor automatically so you don't have to do it manually per project.

Although `prettier` formats code, it doesn't configure the editor, so if the editor is set to 4-space indent, and prettier is set to 2-space indent, then indenting will automatically happen in the editor using 4 spaces, and when we run prettier it will re-format back to 2 spaces, which can be jarring.

And for example when prettier is set to 4-space indent but the editor is configured to 2-space indent, then we will see extra indent guide lines in the editor (f.e. in VS Code).

So having editorconfig is convenient to ensure people's editors have the correct settings, even if otherwise tests would still pass due to ESLint and Prettier formatting the code.